### PR TITLE
Update UG and minor bug fixes

### PR DESCRIPTION
**Pull request by Chongjx**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -88,24 +91,25 @@ Adds a new note to the list of note items (think of it as a notebook).
 
 Format: `add-n /t TITLE [/tag TAG_1] [/tag TAG_2]... [/pin ISPIN] [/archive ISARCHIVE]`
 
+💡 Each note has to have a unique title and it is case-insensitive.
```
Comment (R-Ramana) [inline docs/UserGuide.md:94]:
```suggestion
💡 Each note has to have a **UNIQUE** title and it is **CASE-INSENSITIVE**.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -398,10 +409,10 @@ Expected output:
 ### <a id="add-e"><ins>3.16 Add Event:</ins> `add-e`</a>
 Adds an event to the list.
 
-Format: `add-e /t TITLE /timing DATETIME [/repeat REPEAT] [/stop REPEAT_END] [/remind REMIND]`
+Format: `add-e /t TITLE /timing DATETIME [/repeat REPEAT] [/stop REPEAT_END] [/remind REMIND] [/tag TAG_1] [/tag TAG_2]...`
 
-💡 DATETIME format pattern "dd-MM-yyyy HH:mm:ss”
-💡 Specifying  [/repeat RECURRING] will set the event as a recurring event.
+💡 DATETIME format pattern "dd-MM-yyyy HH:mm:ss” <br>
+💡 Specifying [/repeat RECURRING] will set the event as a recurring event. <br>
```
Comment (R-Ramana) [inline docs/UserGuide.md:415]: Not sure if the line break is necessary, don't think some of it has? Should probably standardize

File: src/main/java/seedu/notus/data/notebook/Notebook.java [hunk 3]
```diff
@@ -247,6 +247,16 @@ public boolean getNote(String noteTitle) {
...
+    /**
+     * Returns true if there is a matching note in the archived notes based on the note title parameter.
+     *
+     * @param noteTitle String containing the title of the note.
+     * @return if the note exist.
+     */
+    public boolean getArchiveNote(String noteTitle) {
+        return archivedNotes.stream().anyMatch(note -> note.getTitle().equalsIgnoreCase(noteTitle));
+    }
+
```
Comment (R-Ramana) [inline src/main/java/seedu/notus/data/notebook/Notebook.java:259]: What is this used for?

**Review by R-Ramana — APPROVED** [review]

File: src/main/java/seedu/notus/data/notebook/Notebook.java (hunk 3 above)
Comment (Chongjx) [reply src/main/java/seedu/notus/data/notebook/Notebook.java:259]: Currently adding of note doesn't check if the title match any archive notes. So I added this. Refer to my changes in AddNoteCommand

Comment (R-Ramana) [reply src/main/java/seedu/notus/data/notebook/Notebook.java:259]: Yea I realized but I forgot to remove this comment! But yeap thanks!

**Pull request merged**